package/utils/visibility.ts

```diff
@@ -0,0 +1,13 @@
+import { Mesh, Object3D, Vector3 } from "three";
+
+export const OOO = new Vector3(0, 0, 0);
+
+export const hideChild = (child: Object3D | Mesh): Object3D | Mesh => {
+    return showChild(child, OOO);
+}
+
+export const showChild = (child: Object3D | Mesh, size: Vector3): Object3D | Mesh => {
+    const { x, y, z } = size;
+    child.scale.set(x, y, z);
```

Comment:
Hier kan je de visible property togglen waardoor de mesh wel of niet gerenderd zal worden. https://threejs.org/docs/#api/en/core/Object3D.visible